Potion.py

```diff
@@ -0,0 +1,63 @@
+class Potion:
+    def __init__(self, name, effect, potency):
+        self.name = name
+        self.effect = effect
+        self.potency = potency
+
+    def use(self):
+        return f"Using {self.name}, which grants {self.effect} with potency of {self.potency}."
+
+
```

Comment:
For all potions where do the level attributes come from? What is it checking against? How do you know if the potion level is lesser? Is it tied to level of player?